script/Arcueid/Day 8/08_00_ARC08_8.txt

```diff
@@ -785,7 +785,7 @@ I could do nothing but stand there in blank amazement.
 [sha:7aa791603cbf625551299e465e7d87f81dd06852]{
 -- Page 526, Offset 21096.
 -- 男はなぎ払われた影と共に、その姿を亡くしていった。
-The man, and the shadow that clung to him, have disappeared.
+The man, along with the shadow that clung to him, has disappeared.
```

Comment:
Would probably reorder this one just so the have/has application isn't as prominent
> The man has disappeared, along with the shadow that had clung to him.